en/lessons/basics/mix-tasks.md

```diff
@@ -13,7 +13,7 @@ It's not uncommon to want to extend your Elixir applications functionality by ad
 Before we learn about how to create specific Mix tasks for our projects, let's look at one that already exists:
 
 ```shell
-$ mix phoenix.new my_phoenix_app
+$ mix phx.new my_phoenix_app
 
 * creating my_phoenix_app/config/config.exs
```

Comment:
@filalex77 The files generated changed a bit with `phx.new`, do you would make sense to update this part as well?

```
* creating my_phoenix_app/config/config.exs                                                                                                                                   
* creating my_phoenix_app/config/dev.exs                                                                                                                                      
* creating my_phoenix_app/config/prod.exs                                                                                                                                     
* creating my_phoenix_app/config/prod.secret.exs                                                                                                                              
* creating my_phoenix_app/config/test.exs                                                                                                                                     
* creating my_phoenix_app/lib/my_phoenix_app/application.ex                                                                                                                   
* creating my_phoenix_app/lib/my_phoenix_app.ex                                                                                                                               
* creating my_phoenix_app/lib/my_phoenix_app_web/channels/user_socket.ex   
```